benches/nosqlbench/tnt_hash.lua

```diff
@@ -0,0 +1,30 @@
+local console = require('console')
+console.listen("/tmp/tarantool-server.sock")
+
+box.cfg {
+    pid_file   = "./tarantool-server.pid",
+    log        = "./tarantool-server.log",
+    listen = 3301,
+    memtx_memory = 2000000000,
+    background = true,
+    checkpoint_interval = 0,
+    wal_mode = 'none',
```

Comment:
Please, provide a reason, why `wal_mode` is used here.

Reply:
The same answer as [here](https://github.com/tarantool/bench-run/pull/14#discussion_r546418427)